### PR TITLE
An open letter to conference organizers

### PR DESCRIPTION
**Pull request by benbalter**

### CODE REVIEW

File: _posts/2014-11-15-an-open-letter-to-conference-organizers.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### 6. Offer scholarships on a needs basis
+
+There's a broader discussion to be had about equality of opportunity in the technology community, but one thing's for sure, at $800+ a pop for the badge alone, lots of smart people who have lots to contribute can't attend. Rather than giving sponsors dozens of conference passes they're not going to use, swap a few out for sponsorship slots to support a local startup or fly a student across the country. GitHub's done this frequently, and it's some of the best sponsorship dollars we've spent, IMHO.
+
+### 7. Scan groceries, not humans
```
Comment (konklone) [inline _posts/2014-11-15-an-open-letter-to-conference-organizers.md]: Looking forward to the contents of this one

**Comment by benbalter** [comment]
That time were you just up and forget to publish a post

**Comment by benbalter** [comment]
@bkeepers, @nacin, @johndbritton, @amateurhuman @bdemott Found this old post from last November that I realized I forgot to publish. As frequent conference goers, any thing else you'd add to this list?

File: _posts/2015-12-07-an-open-letter-to-conference-organizers.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### 6. Offer scholarships on a needs basis
+
+There's a broader discussion to be had about equality of opportunity in the technology community, but one thing's for sure, at $800+ a pop for the badge alone, lots of smart people who have lots to contribute can't attend. Rather than giving sponsors dozens of conference passes they're not going to use, swap a few out for sponsorship slots to support a local startup or fly a student across the country. GitHub's done this frequently, and it's some of the best sponsorship dollars we've spent, IMHO.
+
+### 7. Scan groceries, not humans
```
Comment (bdemott) [inline _posts/2015-12-07-an-open-letter-to-conference-organizers.md]: Nothing I can add but I feel like #_7 was written to conference sponsors and not conference organizers.

Comment (benbalter) [inline _posts/2015-12-07-an-open-letter-to-conference-organizers.md]: Ironically, it was the only one not written out a year ago, and you're 100% right. Going to :skull: it.

**Comment by nacin** [comment]
For conference badges, a specific recommendation I'd make is to render the last name the same size as the first name. Usually when they're different sizes, in order to not look badly designed, they end up being drastically different sizes. Thus while the _first_ name is perhaps legible, you end up being unable to read the last name. This affects me in part because (and I am not alone here) I am basically mononymous, but even when that's not the case, I'm often trying to remember if you're _that_ Ben that I talk with online sometimes, or if I just didn't remember your face correctly. It's just silly to not make the whole thing the same size.

File: _posts/2015-12-07-an-open-letter-to-conference-organizers.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: "An open letter to technology conference organizers"
+excerpt: "Six things I wish every technology conference organizer knew."
+---
+
+I've probably been to more than a hundred technology conferences over the past few years. Some were great, but not all were good. Here's six things I wish every technology conference organizer knew:
+
+### 1. Bulletproof wifi is non-negotiable
+
+If you're a technology conference, you can safely assume most talks are going to involve live demos, participants are going to want to Tweet, Myspace, or whatever the talk, and in between sessions, conference goers still have a day job to which their responsible. If the wifi's slow or unreliable, none of that can happen. The venue's going to say their single Linksys router is the best one Best Buy had on sale, but they're lying. Assume 2-3 devices per participants, none of which have an ethernet port, and plan accordingly.
```
Comment (johndbritton) [inline _posts/2015-12-07-an-open-letter-to-conference-organizers.md:10]: Unless you're actively discouraging internet use. I've been to events with no internet on purpose and it's quite refreshing.

File: _posts/2015-12-07-an-open-letter-to-conference-organizers.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+### 3. Outlets, outlets everywhere
+
+We're geeks. We've got a million things with screens, and we're going to spend all day on the darn things. Hotels and other conference venues strategically place outlets along walls for two purposes: vacuuming and quad-box drops to support head tables in various room arrangements. Conference goers should sit where they want to sit, not on the floor or against the wall. Power strips and extension chords are cheap, but go a long way to freeing conference goers to worry about the conference, not when they're going to get their next ion fix.
+
+### 4. Record every talk
+
+The people in the room are only a small subset the audience for a given talk. The biggest impact the talk will have is when conference goers return home and talk about what they learned. Not to mention, conferences are expensive, and not everyone can be there in person. For the speaker, that means their ideas have a larger reach. For the conference, that means their branding reaches more eyeballs, as they show off what great speakers they attracted, and serves as free advertising for following years.
```
Comment (johndbritton) [inline _posts/2015-12-07-an-open-letter-to-conference-organizers.md:22]: A corollary is to post the videos right away so attendees can share while the memories are fresh.

**Comment by johndbritton** [comment]
Seems reasonable to me, there are a lot more opinions that could be in here, but I think these are mostly facts.

**Pull request closed without merging**